lib/platform/github/massage-markdown-links.ts

```diff
@@ -0,0 +1,55 @@
+import findAndReplace from 'mdast-util-find-and-replace';
+import remark from 'remark';
+import type { Plugin, Transformer } from 'unified';
+// eslint-disable-next-line import/no-unresolved
```

Comment:
`@types/unist` exists in NPM while `unist` doesn't

Reply:
It's a type only package 🤔